src/scss/components/_logosSection.scss

```diff
@@ -0,0 +1,76 @@
+.aliados-wrapper {
```

Comment:
I would suggest using english names in general. I think "sponsors" is a good word instead of "aliados"

Reply:
Also I would recommend to change the filename to _sponsorsSection.scss for consistency